js/i18n/elfinder.uk.js

```diff
@@ -245,8 +245,8 @@ if (elFinder && elFinder.prototype && typeof(elFinder.prototype.i18) == 'object'
 			'moveFiles'       : 'Перемістити файли',
 			'copyFiles'       : 'Копіювати файли',
 			'rmFromPlaces'    : 'Вилучити з розташувань',
-			'untitled folder' : 'неназвана папка',
-			'untitled file.txt' : 'неназваний файл.txt',
+			'untitled folder' : 'Нова папка',
+			'untitled file.txt' : 'new.php',
```

Comment:
Extension of the text file is generally a ".txt". If you wish the ".php", please be overwritten in the following manner.

```html
<script src="js/i18n/elfinder.uk.js"></script>
<script>
(function(){
	var mes_uk;
	if (typeof elFinder.prototype.i18.uk != "undefined") {
		mes_uk = elFinder.prototype.i18.uk.messages;
		mes_uk['untitled file.txt'] = 'new.php'; // over write
		// and more...
	}
})();
</script>
```